lib/rules/no-duplicate-attributes.js

```diff
@@ -37,27 +37,48 @@ function getName (attribute) {
  * @returns {Object} AST event handlers.
  */
 function create (context) {
-  const names = new Set()
+  const options = context.options[0] || {}
+  const allowCoexistStyle = Boolean(options.allowCoexistStyle !== false)
+  const allowCoexistClass = Boolean(options.allowCoexistClass !== false)
```

Comment:
I think that this cast is redundant since the result of `!==` expression is always boolean.